warehouse/models/mart/gtfs_quality/fct_daily_service_combined_guideline_checks.sql

```diff
@@ -17,6 +17,7 @@ fct_daily_service_combined_guideline_checks AS (
         feature,
         check,
         {{ guidelines_aggregation_logic() }} as status,
+        {{ guidelines_aggregation_logic_reports }} as reports_status,
```

Comment:
I think you need parentheses here, mirroring the call on the line above?

Reply:
yes! darnit, I already caught this but I guess I didn't push it

Reply:
Weird, this commit did go thorugh yesterday, yet you still saw the old version.  🤷